tools/agents/gen-manifest.py

```
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+"""מייצר את החלקים של providers.json שיש להם מקור אמת אחר.
+
+**הבעיה.** ‏providers.json נכתב ביד — כלומר מתיישן בשקט, בדיוק כמו
+זמני המודלים אחרי החלפת חומרה. ‏`in_use_by` הוא הדוגמה: שלושת
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)